.github/ISSUE_TEMPLATE/marketing-request.yml

```diff
@@ -0,0 +1,11 @@
+name: Contributor Comms Request
```

Comment:
```
<!-- Want to learn more about Contributor Comms? 

We started the Upstream Marketing Team to focus on contributor-to-contributor communication throughout Kubernetes.

Read about our charter here: https://github.com/kubernetes/community/blob/master/communication/marketing-team/CHARTER.md

Or about the team: https://github.com/kubernetes/community/blob/master/communication/marketing-team

And thanks for reaching out! We're here to help. -->
```

We have comments as such in the Contributor Comms request template. Don't we need it here in the issue forms as well?

cc: @mrbobbytables

Reply:
I think we can omit this these days The charter and team list isn't super useful. A follow up from the marketing team would be a potential list of services if the person wants to know more